Mathlib/Topology/Algebra/UniformField.lean

```diff
@@ -177,6 +177,28 @@ instance : TopologicalDivisionRing (hat K) :=
 
 end Completion
 
+/-- The pullback of a completable topological field along a ring homomorphism
+is a completable topological field. -/
+theorem comap_completableTopField {α β : Type*}
+    [Field β] [Field α] {f : α →+* β} [b : UniformSpace β]
+    [@T0Space α (UniformSpace.comap f b).toTopologicalSpace] [CompletableTopField β] :
+    @CompletableTopField _ _ (UniformSpace.comap f b) := by
+  have h_ui : @UniformInducing _ _ (UniformSpace.comap f b) _ f := by
```

Comment:
You can also
`have inst := UniformSpace.comap f b` in the proof to get rid of the `@`s.

Reply:
Done.